sdk/core/src/mock/mock_request.rs

```diff
@@ -8,17 +8,6 @@ use std::str::FromStr;
 
 const FIELDS: &[&str] = &["uri", "method", "headers", "body"];
 
-impl Request {
-    fn new(uri: Uri, method: Method, headers: HeaderMap, body: Body) -> Self {
-        Self {
-            uri,
-            method,
-            headers,
-            body,
-        }
-    }
-}
-
```

Comment:
Oh, it took me a sec to realize this had just been moved to `core/src/request.rs`. That seems good.